reference/providers/ecs.md

```diff
@@ -7,13 +7,41 @@ sidebar:
 
 {% include toc %}
 
-## Resource Mapping
+## Resource mapping
 
 ### Server Group
 A Spinnaker **server group** maps to an [Amazon ECS service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html#service_concepts).
 
+There are two ways to specify server group settings for ECS:
+
+* __Inputs (default)__: 
+  Specify the container image, resource limits, and other settings needed to create an [ECS task definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) and [service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html#service_concepts) directly in the server group. This option supports deploying one container per Amazon ECS task (equivalent to a Spinnaker "instance").
```

Comment:
Double check on ECS -> Amazon ECS everywhere, to match our naming in the official AWS docs